tests/cache/sorted_set.rs

```diff
@@ -1,78 +1,139 @@
+            elements.sort_by(sort_by_score);
+            SortedSetFetch::Hit {
+                elements: SortedSetElements { elements },
+            }
+        }
+        _ => sorted_set_fetch_result,
+    };
+
+    let mut expected = expected;
+    expected.sort_by(|a, b| {
+        a.1.partial_cmp(&b.1)
+            .expect("expected elements to be sortable")
+    });
+    assert_fetched_sorted_set_eq(sorted_set_fetch_result, expected).await
+}
 
+mod sorted_set_fetch_by_rank {
     use super::*;
 
     #[tokio::test]
     async fn happy_path() -> MomentoResult<()> {
         let client = &CACHE_TEST_STATE.client;
         let cache_name = &CACHE_TEST_STATE.cache_name;
-        let sorted_set_name = unique_key();
-        let to_put = vec![
-            ("1".to_string(), 0.0),
-            ("2".to_string(), 1.0),
-            ("3".to_string(), 0.5),
-            ("4".to_string(), 2.0),
-            ("5".to_string(), 1.5),
-        ];
+        let item = TestSortedSet {
+            name: unique_key(),
+            elements: vec![
+                ("1".to_string(), 0.0),
+                ("2".to_string(), 1.0),
+                ("3".to_string(), 0.5),
+                ("4".to_string(), 2.0),
+                ("5".to_string(), 1.5),
+            ],
+        };
 
         let result = client
-            .sorted_set_fetch_by_rank(cache_name, sorted_set_name.clone(), Ascending, None, None)
+            .sorted_set_fetch_by_rank(cache_name, item.name(), Ascending, None, None)
             .await?;
         assert_eq!(result, SortedSetFetch::Miss);
 
         client
-            .sorted_set_put_elements(cache_name, sorted_set_name.clone(), to_put)
+            .sorted_set_put_elements(cache_name, item.name(), item.elements().to_vec())
             .await?;
 
         // Full set ascending, end index larger than set
-        let fetch_request = SortedSetFetchByRankRequest::new(cache_name, sorted_set_name.clone())
+        let fetch_request = SortedSetFetchByRankRequest::new(cache_name, item.name())
             .order(Ascending)
             .start_rank(0)
             .end_rank(6);
-
         let result = client.send_request(fetch_request).await?;
-
-        match result {
-            SortedSetFetch::Hit { elements } => {
-                assert_eq!(elements.len(), 5);
-                let string_elements: Vec<String> =
-                    elements.into_strings()?.into_iter().map(|e| e.0).collect();
-
-                assert_eq!(string_elements, vec!["1", "3", "2", "5", "4"])
-            }
-            _ => panic!("Expected SortedSetFetch::Hit, but got {:?}", result),
-        }
+        assert_fetched_sorted_set_eq(
+            result,
+            vec![
+                ("1".to_string(), 0.0),
+                ("3".to_string(), 0.5),
+                ("2".to_string(), 1.0),
+                ("5".to_string(), 1.5),
+                ("4".to_string(), 2.0),
+            ],
+        )
+        .await?;
```

Comment:
The tests are much easier to follow after adding `From` and the helper function.